.travis.yml

```diff
@@ -1,52 +1,42 @@
-########################################################################
-# This pipeline is executed on PHP 7.4:
-# - required for development
-# - required to run Rector, to generate the PHP 7.1-compatible code for production
-#
-# It uses the Build Environment "Xenial", because it has PHP 7.1 preinstalled,
-# so we can switch from the global PHP 7.4 to local PHP 7.1 through `phpenv`
-#
-# There are 2 parallel executions, with env variable:
-# 1. DOWNGRADE_CODE="true"
-# 2. DOWNGRADE_CODE="false"
-#
-# When DOWNGRADE_CODE is "false", it is the standard testing pipeline:
-# Execute PHPCS, PHPUnit and PHPStan to check there are no issues, and generate report.
-#
-# When DOWNGRADE_CODE is "true", it tests the conversion of the PHP code version 7.4 to 7.1:
-# - Run Rector to downgrade the code
-# - Run PHPStan. If it exits with no errors, then downgrading the code works well.
-########################################################################
```

Comment:
Please don't remove the documentation. I want to explain the objective of the PHP 7.1 job (previously through env var `DOWNGRADE`CODE`)

Reply:
Done.